index.html

```diff
@@ -0,0 +1,147 @@
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/energy.png" alt="clean-energy"/>
+                <p class="first-p">Energy</p>
+                <p class="second-p">Exploration of energies that do not pollute environment such as biogas.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/plant.png" alt="conserve plants"/>
+                <p class="first-p">Forest</p>
+                <p class="second-p">Planting trees is very important in conservation of animal habitat and others that will be discussed.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/light.png" alt="Knowledge"/>
+                <p class="first-p">Knowledge</p>
+                <p class="second-p">Importance of teaching our young ones the benefit of conserving environment.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/recycle.png" alt="Recycle"/>
+                <p class="first-p">Recycle</p>
+                <p class="second-p">Plastic waste contribute to a huge percent of sea animals suffocation.</p>
+            </div>
+        </div>
+        <div class="grid-item2">
+            <h3 class="join-p">Join Green Earth Global Summit</h3>
+        </div>
+
+    </div>
+    <h3 class="heading31">SEE THE WHOLE PROGRAM</h3>
+    <hr class="hrprog">
+   </section>
+
+   <!-- Featured speakers -->
+   <section class="Featured">
+  <h4 class="speakers">Featured Speakers</h4>
+  <hr class="hr">
+  <div class="professor-cards" id="professor-cards">
+    <!-- generated when index.js loads -->
+  </div>
+  <button type="button" class="more-professors-btn" id="more-professors-btn">More<img src="assets/arrow_down.png" id="more" alt="more"/>
+  </button>
+   </section>
+
+   <!-- partners -->
+  <section class="partners">
+    <h5 class="partners-heading">Partner</h5>
+    <span class="hr"></span>
+    <ul class="partner-icons-list">
+      <ul class="partner-icons">
+        <li><img class="partner-logo" src="assets/mozilla.png" alt="mozilla"/></li>
+        <li><img class="partner-logo" src="assets/googlekorea.png" alt="google"/></li>
+        <li><img class="partner-logo" src="assets/naver.png" alt="naver"/></li>
+     
+      </ul>
+      <ul class="partner-icons">
+        <li><img class="partner-logo" src="assets/daumkakaologo.png" alt="kakao"/></li>
+        <li><img class="partner-logo" src="assets/airbnb.png" alt="airbnb"/></li>
+      </ul>
+    </ul>
+  </section>
+  <!-- footer -->
+  <footer class="footer" id="footer-about">
+    <div class="footer-logo-container"><img  class="footer-logo" id="earth" src="assets/Earth.png" alt="logo"/>
+   
+    <p class="copyright">2023 Green Earth Global. <br> Some Rights Reserved.</p></div> 
+  </footer>
```

Comment:
- [ ] Please ensure that the design of the footer matches the original design. You should make sure that the background color is white and also the size should be the same as the one of the original design ☺️.

Your design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/7a06e4bc-0256-4481-b4f5-f90fff0d2904)

Original design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/338a885b-c1be-4606-8326-69e45f682ca6)